pkg/datarepair/checker/checker.go

```diff
@@ -146,3 +165,15 @@ func lookupResponsesToNodes(responses *pb.LookupResponses) []*pb.Node {
 	}
 	return nodes
 }
+
+func combinedError(errs []error) error {
```

Comment:
we are using combined error in a couple places in code now. Each place has it's own combined error. Move this function definition to the utils folder

Reply:
There's already `utils.CombineErrors`